manifests/ruby.yml

```diff
@@ -355,8 +355,6 @@ tests/:
       Test_Trace_Sampling_Tags: missing_feature
       Test_Trace_Sampling_Tags_Feb2024_Revision: missing_feature
       Test_Trace_Sampling_With_W3C: missing_feature
-    test_tracer.py:
-      Test_TracerSCITagging: missing_feature
```

Comment:
can we find the version that this was implemented in? (same motivation as having the go version)